src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -1435,20 +1435,21 @@ public override BoundNode VisitIndexerAccess(BoundIndexerAccess node)
 
         public override BoundNode VisitIndexOrRangePatternIndexerAccess(BoundIndexOrRangePatternIndexerAccess node)
         {
-            // Index or Range pattern indexers evaluate the following in order:
+            // Index or Range implicit indexers evaluate the following in order:
             // 1. The receiver
-            // 1. The Count or Length method off the receiver
-            // 2. The argument to the access
-            // 3. The pattern method
+            // 2. The Count or Length method off the receiver
+            // 3. The argument to the access
```

Comment:
> 3. The argument to the access [](http://example.com/codeflow?start=14&length=30)

Does this follow the current behavior? Shouldn't the argument be evaluated right after the receiver?